README.md

```diff
@@ -1,3 +1,186 @@
 # RemoteResourceS3
 
 [![Build Status](https://travis-ci.com/razee-io/RemoteResourceS3.svg?branch=master)](https://travis-ci.com/razee-io/RemoteResourceS3)
+![GitHub](https://img.shields.io/github/license/razee-io/RemoteResourceS3.svg?color=success)
+
+RemoteResourceS3 is a variant of RemoteResource. RemoteResource is the foundation
+for implementing continuous deployment with kapitan. It retrieves and applies the
+configuration for all resources. RemoteResourceS3 extends that functionality by
+easily authenticate with S3 object storages.
```

Comment:
`RemoteResourceS3 extends that functionality by easily authenticate with S3 object storages. `
becomes
`RemoteResourceS3 extends that functionality by supporting authentication to object storage services that implement the S3 interfaces.`